webapp/src/blocks.tsx

```diff
@@ -882,6 +915,9 @@ export class Editor extends toolboxeditor.ToolboxEditor {
             <div className="blocksAndErrorList">
                 <div className="blocksEditorOuter">
                     <div id="blocksEditor"></div>
+                    {/* This does not integrate well with MakeCode, but is useful for blockly
+                     keyboard navigation testing / evaluation purposes. */}
+                    <div id="shortcuts"></div>
```

Comment:
Remove this div as we don't intend to use its contents as populate by the keyboard plugin

Reply:
Removed here [d0d6bb0](https://github.com/microsoft/pxt/pull/10512/commits/d0d6bb0c2dd1b8d8aaf73ce794f9e2d8ca54d03d)